scripts/cpg_cromwell/export_analysis_outs.py

```diff
@@ -0,0 +1,29 @@
+"""
+Move the CHIP pipeline outputs to the release bucket for export to the GML private buckets
+"""
+import hailtop.batch as hb
+from cpg_utils.config import get_config
+from cpg_utils.hail_batch import remote_tmpdir, get_batch
+
+import re
+
+_config = get_config()
+DATASET = _config['workflow']['dataset']
+JOB_NAME = _config['workflow']['name']
+WORKFLOW_NAME = _config['workflow']['workflow_name']
+RUN_ID = _config['workflow']['run_id']
+OUTPUT_PREFIX = f'{JOB_NAME}/{WORKFLOW_NAME}/{RUN_ID}'
+CROMWELL_OUTPUT_PREFIX = f'mutect2-chip/{OUTPUT_PREFIX}'
+CROMWELL_OUTPUT_BUCKET = _config['storage'][DATASET]['default']
+CROMWELL_OUTPUT_PATH = f'{CROMWELL_OUTPUT_BUCKET}/{CROMWELL_OUTPUT_PREFIX}'
+# OUTPUT_BUCKET = _config['storage'][DATASET]['release']  # Currently no 'release' key in config TOML
+OUTPUT_BUCKET = re.sub(r'\-main$', '-release', CROMWELL_OUTPUT_BUCKET)
+OUTPUT_PATH = f'{OUTPUT_BUCKET}/{CROMWELL_OUTPUT_PREFIX}'
+
+b = get_batch()
+
+process_j = b.new_job('move-chip-output-files')
+
+process_j.command(f"gcloud -q auth activate-service-account --key-file=/gsa-key/key.json; gsutil mv {CROMWELL_OUTPUT_PATH} {OUTPUT_PATH}")
```

Comment:
@illusional will this work? From memory writing data to release should be fine, for reading the billing project will have to be set specifically in the command

Reply:
Writing is a Class A operation, so pretty sure it requires a billing account:

<img width="886" alt="image" src="https://github.com/populationgenomics/cromwell-kccg-mutect2-chip/assets/22381693/362e115b-ec3d-4d66-a0c9-34e04feccbf8">